build/configs/rtl8721csm/security_hal_test_tz/Make.defs

```diff
@@ -27,6 +27,11 @@ else
 LDSCRIPT = rlx8721d_img2_is.ld
 endif
 
+ifeq ($(CONFIG_AMEBAD_TRUSTZONE),y)
+EXTRA_LIBS += ${TOPDIR}/board/rtl8721csm/src/libs/cmse_implib.a
```

Comment:
This was added at https://github.com/Samsung/TizenRT/blob/master/os/arch/arm/src/amebad/Make.defs#L173 so that this is duplicated. Let's remove.

Reply:
Sorry, my mistake, I did not sync with other Make.defs.
Removed